src/components/gateway/GatewayCreatePage.tsx

```
@@ -1517,42 +1533,56 @@ const GatewayCreatePage: React.FC = () => {
+                  validation={formValidation()}
+                  model={gatewayModel}
+                  resource={gatewayObject}
+                  policyType="Gateway"
+                  navigate={navigate}
+                  redirectPath={`/k8s/ns/${selectedNamespace}/${gatewayModel?.apiGroup}~${gatewayModel?.apiVersion}~${gatewayModel?.kind}/${gatewayName}`}
+                />
+                <Button
+                  variant="link"
+                  onClick={() => navigate(-1)}
+                  style={{ marginLeft: '24px' }}
+                >
```

ESLint

[error] Replace `⏎··················variant="link"⏎··················onClick={()·=>·navigate(-1)}⏎··················style={{·marginLeft:·'24px'·}}⏎················` with `·variant="link"·onClick={()·=>·navigate(-1)}·style={{·marginLeft:·'24px'·}}`

(prettier/prettier)